samples/python/stitching_detailed.py

```diff
@@ -379,9 +394,14 @@
             fixedFileName = img_names[idx][:pos_s + 1 ]+"fixed_" + img_names[idx][pos_s + 1: ]
         cv.imwrite(fixedFileName, timelapser.getDst())
     else:
-        blender.feed(image_warped_s, mask_warped, corners[idx])
+        blender.feed(cv.UMat(image_warped_s), mask_warped, corners[idx])
 if not timelapse:
     result=None
     result_mask=None
     result,result_mask = blender.blend(result,result_mask)
-    cv.imwrite(result_name,result)
+    zoomx =600/result.shape[1]
+    dst=cv.normalize(src=result,dst=None,alpha=255.,norm_type=cv.NORM_MINMAX,dtype=cv.CV_8U)
+    dst=cv.resize(dst,dsize=None,fx=zoomx,fy=zoomx)
+    cv.imwrite(result_name,dst)
```

Comment:
I believe we should write into file non-resized (original) result.
For `imshow()`, resize is fine.

Reply:
OK